lumicks/pylake/tests/test_kymotracker.py

```diff
@@ -404,3 +404,13 @@ def test_kymolines():
 
     with pytest.raises(TypeError):
         lines.extend(5)
+
+
+def test_filter_lines():
+    k1 = KymoLine([1, 2, 3], [1, 2, 3])
+    k2 = KymoLine([2, 3], [1, 2])
+    k3 = KymoLine([2, 3, 4, 5], [1, 2, 4, 5])
+    lines = KymoLines([k1, k2, k3])
+    assert len(filter_lines(lines, 5)) == 0
+    assert len(filter_lines(lines, 3)) == 2
+    assert len(filter_lines(lines, 2)) == 3
```

Comment:
I don't think it's enough to test that `filter_lines` is doing it's job right. I'd rather check the actual values instead of `len`.

Reply:
Yes. Amending! :)